src/screens/ProfileScreen/elements/FollowInfoRow.js

```diff
@@ -60,12 +60,12 @@ let styles = StyleSheet.create({
   following: {marginLeft: 18},
   textTotal: {
     fontSize: 14,
-    color: colors.bondi_blue,
+    color: COLORS.signed_primary,
     paddingRight: 4
   },
   textFollow: {
     fontSize: 14,
-    color: colors.black,
+    color: COLORS.black,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color values for `textTotal` and `textFollow` have been updated to use the new `COLORS` object. The color value for `textTotal` has changed from `colors.bondi_blue` to `COLORS.signed_primary`, and the color value for `textFollow` remains `COLORS.black`. Ensure that these new color values are consistent with the intended design.
```diff
-    color: colors.bondi_blue,
+    color: COLORS.signed_primary,

-    color: colors.black,
+    color: COLORS.black,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->